changelog.md

```diff
@@ -1,3 +1,107 @@
+<a name="6.1.0"></a>
+# 6.1.0 (2020-06-25)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.0.0...v6.1.0)
+
+We expect this release to ship in the DevTools of [Chrome 85](https://chromiumdash.appspot.com/schedule).
+
```

Comment:
should we also be saying "we expect this release to ship in PageSpeed Insights in a week"?

Reply:
cc @exterkamp

Reply:
I would say "Within 2 weeks" just to give us some runway if we end up needing to debug something/have weird issues.